### DISCUSSION TITLE
LazyVim does not search gitignore files by default, how can I use telescope to search for keywords in node_modules folder?

### DISCUSSION THREAD

**Discussion by xclidongbo** [LazyVim/LazyVim#1177] [General]
LazyVim does not search gitignore files by default, how can I use telescope to search for keywords in node_modules folder?

**Comment by dpetka2001** [comment]
How about `Telescope find_files no_ignore=true`?

**Reply by xclidongbo** [reply]
No，grep_string, not find_files.

**Reply by dpetka2001** [reply]
You might want to install this [plugin](https://github.com/nvim-telescope/telescope-live-grep-args.nvim). It allows you to pass arguments to your live-grep searches. For instance, you can pass `--no-ignore` to disable `.gitignore` filtering.

**Reply by pprotas** [reply]
What would be the best way to replace the regular Live Grep with this plugin? So the `<leader>/` and `<leader>sg` shortcuts?

**Reply by dpetka2001** [reply]
You should the plugin's README. I don't use it myself. Just found it while searching around.

**Reply by pprotas** [reply]
The readme doesn't contain any LazyVim-specific configuration examples, so that information is a bit hard to find. 

I got the plugin working quite easily, but just re-binding the Telescope shortcuts feels "wrong" because LazyVim uses `Util.telescope`

**Reply by dpetka2001** [reply]
Just make a new keybinding that is not used by LazyVim. Although I'm sure you can still use `Util.telescope` to achieve the same behavior.
